src/app/app.module.ts

```diff
@@ -188,6 +195,7 @@ registerLocaleData(localeZhTw, 'zh-TW');
     ],
     entryComponents: [
         AttachmentsComponent,
+        ChangePasswordComponent,
```

Comment:
Should not need these in here since these are not modals.

Reply:
Fixed